Include AF option in diagnosing problems page (#926)
zaproxy/zaproxy-website: site/content/docs/docker/diagnosing-problems.md

   ```

All of the requests and responses made by or proxied through ZAP will be written to the `req-resp-log.txt` file in your CWD as above.

### Automation Framework

If you are using the Automation Framework then you will need to:

1. Copy the following file to your CWD
   * [LogMessages.js](https://github.com/zaproxy/community-scripts/blob/main/httpsender/LogMessages.js)

> 2. Modify your YAML plan file to include this script as a parameter, e.g.

Reviewer: ```suggestion
2. Modify your YAML plan file to add the script, e.g.
```
